Introducing versioning in descheduler

ravisantoshgudimetla opened the pull request:
Signed-off-by: ravisantoshgudimetla <ravisantoshgudimetla@gmail.com>

/cc @aveshagarwal. For versioning.

ravisantoshgudimetla commented:
Addresses #24.

aveshagarwal commented on cmd/descheduler/app/version.go:
```diff
@@ -0,0 +1,37 @@
+)
+
+// This becomes easy for exporting later. In fact, this could be moved to top level directory.
+const DeschedulerVersion = "0.1.0"
```
Instead of hard coding, could you check how to print something like, for example, last tag+commits since last tag or something similar. You could also see how it is done in kube/openshift, and have something similar.

ravisantoshgudimetla replied:
Sure, I initially wanted to do that, looking at the kube code but thought it was unnecessary at this stage but I can.

aveshagarwal replied:
actually hard coding 0.1.0 might give wrong information even at this stage, and also it would require updating whenever we have a new release with a new tag.

ravisantoshgudimetla commented:
/cc @aveshagarwal Addressed the comments. Now, the output would be similar to kubectl version. Sample output:

$_output/bin/descheduler version
`Descheduler version {Major:0 Minor:2+ GitCommit:9f448ce12756bb0b9d71d82c50b04e66a91237df GitVersion:v0.2.0-2-g9f448ce BuildDate:2017-10-24T11:07:42-0400 GoVersion:go1.8.3 Compiler:gc Platform:linux/amd64}`
